monochart/values.yaml

```diff
@@ -451,6 +451,7 @@ persistence:
 
 # Ingress for load balancer
 ingress:
+  className: nginx
```

Comment:
I forget, does values.yaml here act as a set of defaults? Or just reference?

If this is a default, I'm not sure if we're using nginx or ingress-nginx.